test/game_teams_test.rb

```diff
@@ -1,11 +1,8 @@
-require 'simplecov'
-SimpleCov.start
-require 'minitest/autorun'
-require 'minitest/pride'
-require 'mocha/minitest'
+require './test/test_helper'
 require './lib/game_teams'
```

Comment:
Wondering whether there's value in requiring all lib files at the test helper level? Would obviously require some unnecessarily at certain points, but would also ensure they're all always there - let's discuss during work time today. I can see pros/cons of either, unclear on what best practice is.